example/bin/main.dart

```diff
@@ -12,7 +12,8 @@ void main() async {
 
   final butterflyStream = commandBuilder.addAsset("assets/Butterfly-209.mp4");
   final beeStream = commandBuilder.addAsset("assets/bee.mp4");
-  final outputStream = commandBuilder.createStream(hasVideo: true, hasAudio: true);
+  final outputStream =
```

Comment:
Please set line length to 120 and revert any formatting changes due to shorter lines.